samples/emcee/build.gradle.kts

```diff
@@ -1,7 +1,7 @@
 plugins {
     `kotlin-dsl`
-    id("com.android.application") version "7.1.3" apply false
-    id("org.jetbrains.kotlin.android") version "1.5.31" apply false
+    id("com.android.application") version "7.2.1" apply false
+    id("org.jetbrains.kotlin.android") version "1.6.21" apply false
```

Comment:
I'd be better to merge such changes beforehand in a separate PR.
It will help to reduce scope of changes for review.